site/_posts/2018-08-07-0.10.0-release.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Apache Arrow 0.10.0 Release"
+date: "2018-08-07 00:00:00 -0400"
+author: wesm
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 0.10.0 release. It is the
+product of over 4 months of development and includes [**470 resolved
+JIRAs**][1]. It is the largest release so far in the project's history. 90
```

Comment:
Write "issues" instead of "JIRAs"? Not everybody knows (or even cares) that JIRA is our issue tracker.

Reply:
done